extension/content.js

```diff
@@ -201,11 +201,25 @@ function showRecentlyPushedBranches() {
 		return;
 	}
 
-	const uri = `/${repoUrl}/show_partial?partial=tree/recently_touched_branches_list`;
-	const fragMarkup = `<include-fragment src=${uri}></include-fragment>`;
-	const div = document.createElement('div');
-	div.innerHTML = fragMarkup;
-	$('.repository-content').prepend(div);
+	const codeURI = $('[data-hotkey="g c"]').attr('href');
```

Comment:
We can technically parse this out of the URI, but just went with this to avoid having to muck with that at all. Will change if requested